grpc/nodejs/package.json

```diff
@@ -10,12 +10,13 @@
     "url": "https://github.com/vaticle/typedb-protocol"
   },
   "dependencies": {
-    "@grpc/grpc-js": "1.5.0"
+    "@grpc/grpc-js": "1.8.17",
+    "protoc-gen-ts": "0.8.6"
   },
   "devDependencies": {
-    "@bazel/typescript": "2.3.1",
-    "grpc-tools": "1.11.2",
-    "grpc_tools_node_protoc_ts": "5.3.2",
-    "typescript": "3.9.7"
+    "@bazel/typescript": "5.8.1",
+    "grpc-tools": "1.12.4",
+    "grpc_tools_node_protoc_ts": "5.3.3",
+    "typescript": "4.9.5"
```

Comment:
pin a specific version to because `protoc-gen-ts` has a bug in its typescript version validation: https://github.com/thesayyn/protoc-gen-ts/issues/203